addons/resource/models/resource_calendar.py

```diff
@@ -113,6 +120,28 @@ def _compute_hours_per_day(self):
             attendances = calendar._get_global_attendances()
             calendar.hours_per_day = calendar._get_hours_per_day(attendances)
 
+    @api.depends('attendance_ids.hour_from', 'attendance_ids.hour_to')
+    def _compute_hours_per_week(self):
+        for calendar in self:
+            sum_hours = sum(
+                (a.hour_to - a.hour_from) for a in calendar.attendance_ids if a._is_work_period())
```

Comment:
```suggestion
            sum_hours = sum(
                (attendance.hour_to - attendance.hour_from) for attendance in calendar.attendance_ids if attendance._is_work_period())
```
we try to avoid one letter variable for understanding :)